spalp.module

```diff
@@ -78,10 +78,6 @@ function spalp_get_app_ids(FieldStorageConfig $definition, ContentEntityInterfac
  * Implements hook_modules_installed().
  */
 function spalp_modules_installed($modules) {
-  // TODO: Requires cleanup.
```

Comment:
there's also `system_get_info()`, but at the point of execution, it doesn't include info about the newly installed modules

so I think this might be the only option

Reply:
I havent looked into this too deeply, but happy to follow your judgement.